WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1687,7 +1687,7 @@ private synchronized void updatePostObject(boolean isAutosave) throws EditorFrag
             }
         }
 
-        // only makes sense to change the publish date and locallychanged date if the Post was actaully changed
+        // only makes sense to change the publish date and locally changed date if the Post was actaully changed
```

Comment:
Just noting there's another typo that can be fixed (same line): `s/actaully/actually/`